qutebrowser/browser/adblock.py

```diff
@@ -192,6 +190,22 @@ def adblock_update(self):
                 self._in_progress.append(download)
                 download.finished.connect(
                     functools.partial(self.on_download_finished, download))
+    def _import_local(self, filename):
+        """Adds the contents of a file to the blocklist
+
+        Args:
+            filename: path to a local directory to import.
```

Comment:
That would be a file, not a directory, right?

Reply:
done